### DISCUSSION TITLE
Group consecutive emote messages into one

### DISCUSSION THREAD

**Discussion by doximanman** [Chatterino/chatterino2#3577] [Ideas]
A feature where consecutive emote messages would be grouped into one message. For instance these messages:
person A: LULW
person B: LULW
person C: LULW

would be shown as (assuming LULW is an enabled emote):
LULW x3 combo

destiny.gg has this feature if you wish to see how it is in practice:
![image](https://user-images.githubusercontent.com/56520848/155549324-b8d6c21e-0eaf-4495-b078-247c43ea99e6.png)

I think this would be a cool feature to make faster chats more readable when a ton of people are using the same emotes consecutively.

Note: This is a duplicate of #892 , but that post was closed claiming it was solved by #1496 , but it wasn't. The request is to group multiple emote messages (possibly from different users) into one, it isn't related to one man spamming.

Thanks!

**Comment by doximanman** [comment]
bumping because i think it would be a cool feature, destiny.gg is open source (and has a public git repository with everything), so maybe that makes this easier to implement? 
thanks.

**Reply by pajlada** [reply]
I appreciate the enthusiasm, but bumping discussions or issues without any new relevant information should not be used to bring more attention to the issue as it just creates more spam for us maintainers.

I personally like this feature so I'd support it being added to Chatterino but until someone shows willingness to implement it this is where we're at.

**Reply by doximanman** [reply]
understood. thanks.
